src/abstractWrappers/Notifications.js

```diff
@@ -432,36 +533,59 @@ class Notifications extends AbstractWrapper {
   // *        Helpers           * //
   // **************************** //
   /**
-  * Helper method to create handler with correct params
-  */
-  _createHandler = (handler, arbitratorAddress, account, callback) => {
-    return (
-      args
-    ) => handler(args, arbitratorAddress, account, callback)
-  }
+   * Helper method to create handler with correct params
+   * @param {function} handler - The handler.
+   * @param {string} arbitratorAddress - The arbitratorAddress.
+   * @param {string} account - The account.
+   * @param {function} callback - The callback.
+   * @returns {object} - The created handler.
+   */
+  _createHandler = (handler, arbitratorAddress, account, callback) => args =>
+    handler(args, arbitratorAddress, account, callback)
 
+  /**
+   * Sends a push notification.
+   * @param {function} callback - The callback.
+   * @param {object} notification - The notification.
+   */
   _sendPushNotification = async (callback, notification) => {
     if (callback && notification) {
       callback(notification)
     }
   }
 
-  _createNotification = (notificationType, message, data) => {
-    return {
-      notificationType,
-      message,
-      data
-    }
-  }
+  /**
+   * Creates a notification object.
+   * @param {number} notificationType - The notificationType.
+   * @param {string} message - The message.
+   * @param {object} data - The data.
+   * @returns {function} - The notification object.
+   */
+  _createNotification = (notificationType, message, data) => ({
+    notificationType,
+    message,
+    data
+  })
 
+  /**
+   * Creates a new notification object in the store.
+   * @param {string} account - The account.
+   * @param {string} txHash - The txHash.
+   * @param {number} logIndex - The logIndex.
+   * @param {number} notificationType - The notificationType.
+   * @param {string} message - The message.
+   * @param {object} data - The data.
+   * @param {bool} read - Wether the notification has been read or not.
+   * @returns {function} - The notification object.
+   */
   _newNotification = async (
     account,
     txHash,
     logIndex,
     notificationType,
     message = '',
     data = {},
-    read = false,
+    read = false
```

Comment:
I'm not super picky on this linting rule because I think I am in the minority opinion in general but I prefer having extra commas at the end of every iterable. I think it is a simple way to avoid bugs

Reply:
The linter will catch any missing commas. It's merely a style thing.